examples/build_file_generation/gazelle_python.yaml

```diff
@@ -57,6 +78,23 @@ manifest:
     idna.package_data: idna
     idna.uts46data: idna
     psutil: psutil
+    psutil.tests: psutil
```

Comment:
These should not be here. 🤔 
Did you re-generate this file after rebasing?

Reply:
Yah.  It was weird that it got in.  The rebase was messy btw.  It me get it out.